tests/unit_tests/Request/RequestTest.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Request;
+
+use GuzzleHttp\Psr7\Response;
+use GuzzleHttp\Psr7\Stream;
+use GuzzleHttp\RequestOptions;
+use GuzzleHttp\Psr7\Request as HttpRequest;
+use GuzzleHttp\Client;
+use League\Container\Container;
+use Pantheon\Terminus\Config\TerminusConfig;
+use Pantheon\Terminus\Request\Request;
+use Pantheon\Terminus\Session\Session;
+use Psr\Log\LoggerInterface;
+
+class RequestTest extends \PHPUnit_Framework_TestCase
```

Comment:
It's missing tests for `download` (not currently being used and needs to be made non-static) and `pagedRequest`.